app/views/posts/index.html.erb

```diff
@@ -1,3 +1,48 @@
-<h1>User Posts</h1>
 
-<p>Here is a list of posts for a given user</p>
\ No newline at end of file
+<section>
+
+<div class="user" >
+  <% if @posts[0] != nil %>
+    <% author = @posts[0].author %>
+        <img class="pro-pic" src='<%author.photo%>' alt='Profile-pic'> <br>
+      <div class="user-dets">
+        <h2 class="user-name"><%=author.name%></h2>
+        <p class='num-posts'>Number of Posts: <%=author.posts_counter%></p>
+      </div>
+  <% else %>
+    <p class='user-name'>This User has no posts</p>
+  <%end%>
+</div>
+
+
+<div class="posts">
+  <% @posts.each do |post|%>
+    <div class="post-and-comment" >
+      <div class="post">
+      <% if(@posts != []) %>
+        <p><%= post.title%></p>
+        <p><%= post.id%></p>
```

Comment:
- [ ] Also it could be better to make it navigate to the `show page`, let take an example from this:
````ruby
<div class="posts">
  <% @posts.each do |post|%>
    <a href="/users/<%= post.author.id %>/posts/<%= post.id %>" class="post-and-comment">
      <div class="post">
      <% if(@posts != []) %>
        <p><%= post.title%></p>
        <p><%= post.id%></p>

        <p><%= post.text%><p>
        <div class="comments-likes">
          <p>Comments: <%= post.comments_counter%><p>
          <p>Likes: <%= post.likes_counter%><p>
        </div>
        <% else %>
        <p>No Posts Yet, Add one to see them Here!</p>
        <% end %>
      </div>
      <% if(post.five_most_recent_comments != []) %>
      <div class="post-comments">
        <% post.five_most_recent_comments.each do |comment| %>
        <p><%= comment.author.name%>: <%= comment.text%></p>
        <% end %>
      </div>
      <% end %>
    </a>
  <%end%>
</div>
````

Kindly, fix it 👍

Reply:
Well resolved ✔️